docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -0,0 +1,756 @@
+              physicalItems {
+                name
+                quantity
+              }
+              digitalItems {
+                name
+                quantity
+              }
+              giftCertificates {
+                name
+              }
+              customItems {
+                name
+                quantity
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    ```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "createCartInput": {
+        "lineItems": [
+          {
+            "quantity": 1,
+            "productEntityId": 111
+          }
+        ]
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+    ```json filename="Example mutation: Create a cart (simple)" showLineNumbers copy
+    {
+      "data": {
+        "cart": {
+          "createCart": {
+            "cart": {
+              "entityId": "bb916deb-ddd6-4586-b65b-b8385e0e7a9d",
+              "lineItems": {
+                "physicalItems": [
+                  {
+                    "name": "[Sample] Smith Journal 13",
+                    "quantity": 1
+                  }
+                ],
+                "digitalItems": [],
+                "giftCertificates": [],
+                "customItems": []
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+  </Tab>
+</Tabs>
+
+## Add cart line items
+
+To add a new line item to the existing cart, add the cart ID to the GraphQL variables or the mutation will fail.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Example mutation: Add cart line items" showLineNumbers copy
+    mutation addCartLineItems($addCartLineItemsInput: AddCartLineItemsInput!) {
+      cart {
+        addCartLineItems(input: $addCartLineItemsInput) {
+          cart {
+            entityId
+          }
+        }
+      }
+    }
+    ```
+
+    ```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "addCartLineItemsInput": {
+        "cartEntityId": "bb916deb-ddd6-4586-b65b-b8385e0e7a9d",
+        "data": {
+          "lineItems": [
+            {
+              "quantity": 1,
+              "productEntityId": 107
+            }
+          ]
+        }
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+    
+    ```json filename="Example mutation: Add cart line item" showLineNumbers copy
+    {
+      "data": {
+        "cart": {
+          "addCartLineItems": {
+            "cart": {
+              "entityId": "bb916deb-ddd6-4586-b65b-b8385e0e7a9d"
+            }
+          }
+        }
+      }
+    }
+    ```
+  </Tab>
+</Tabs>
+
+## Get a checkout
+
+The query example below shows how to display the contents of a checkout. 
+
+<Callout type="info">
+You can access Cart and Checkout details in the same request as other information.
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    
+    ```graphql filename="Example query: Get checkout" showLineNumbers copy
+    query getCheckout {
```

Comment:
Another implication of using a customer impersonation token:  This query is going to need an argument with the cart/checkout ID to return the details of a specific checkout.  (Again, it's different than if you were using a standard token where "my storefront session", essentially, is context provided to every GraphQL request.)

On the subject of providing the checkout ID as an argument: We should call out that it's the same value as the cart ID.

Reply:
@bc-traciporter I'm not sure this was fully addressed. The `getCheckout` example query still doesn't show a checkout ID being passed in as an argument.